packages/react/src/shared.tsx

```diff
@@ -179,8 +188,8 @@ export type SharedDependencies = {
   momokaRelayer: MomokaRelayer;
   onChainRelayer: OnChainRelayer;
   profileCacheManager: IProfileCacheManager;
-  providerFactory: ProviderFactory;
   publicationCacheManager: PublicationCacheManager;
+  tokenAvailability: TokenAvailability;
```

Comment:
as you added this to shared, could you clean up the extra instance in useFollowController, https://github.com/lens-protocol/lens-sdk/blob/9dd33b0370283d332402d2313b189b5883a8b3d5/packages/react/src/transactions/adapters/useFollowController.ts#L82